include/boost/geometry/strategy/geographic/area.hpp

```diff
@@ -127,8 +144,14 @@ protected :
         {
             return_type result;
 
-            return_type sum = spheroid_const.m_c2 * m_excess_sum
-                   + spheroid_const.m_e2 * spheroid_const.m_a2 * m_correction_sum;
+            return_type const spherical_term = spheroid_const.m_c2 * m_excess_sum;
+            return_type const ellipsoidal_term = spheroid_const.m_e2
+                * spheroid_const.m_a2 * m_correction_sum;
+
+            // ignore ellipsoidal term if is large (probably from an azimuth
+            // inaccuracy)
+            return_type sum = math::abs(ellipsoidal_term/spherical_term) > 0.01
```

Comment:
Here is another magic number. If it's from azimuth inaccuracy shouldn't it be based on formula?

Reply:
It is independent from the choice of formula, i.e. for an accurate formula this condition should always be false. It describes  how  tolerate we are with errors in inaccurate formulas.